pkgs/applications/editors/notepadqq/default.nix

```
@@ -13,7 +12,7 @@ in stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [
-    pkgconfig which qttools
+    pkgconfig which qttools wrapQtAppsHook
```

```suggestion
    pkgconfig which qttools
```

is added automatically